AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/CrashLoggingOkHttpInterceptorProvider.kt

```diff
@@ -0,0 +1,8 @@
+package com.automattic.android.tracks.crashlogging
+
+import io.sentry.android.okhttp.SentryOkHttpInterceptor
+import okhttp3.Interceptor
+
+object CrashLoggingOkHttpInterceptorProvider {
+    fun createInstance(): Interceptor = SentryOkHttpInterceptor()
```

Comment:
❓ This is not used anywhere, so clients won't be able to use this yet, right? I assume this is planned for the upcoming PRs, right?

Reply:
It's not used anywhere in the lib, but just adding this to the list of interceptors in client will start adding breadcrumbs. Here's snippet of how it'll look on `WCAndroid`: 
<img width="896" alt="image" src="https://user-images.githubusercontent.com/5845095/121145369-2be73680-c83f-11eb-85b0-f73e7d6f03b7.png">